custom_components/affalddk/config_flow.py

```diff
@@ -58,12 +58,13 @@ def _show_setup_form(
         if user_input is None:
             user_input = {}
 
+        options = [key for key, val in MUNICIPALITIES_LIST.items() if val[1] in ['2', '3', '4', '5']]
```

Comment:
remember to activate the municipalities from affafldsportalen
```suggestion
        options = [key for key, val in MUNICIPALITIES_LIST.items() if val[1] in ['2', '3', '4', '5', '6']]
```